packages/devtools-network-console/src/ui/ResponseViewer/index.tsx

```diff
@@ -260,10 +261,12 @@ function mapStateToProps(state: IView, ownProps: IOwnProps): IConnectedProps {
         });
         throw new Error('Invariant failed: Response not found for given request ID');
     }
-
+    const wsState = state.websocket.get(ownProps.requestId);
+    const showWSView = wsState ? wsState.messages.size > 0 : false
```

Comment:
I guess this makes sense (the first message will be a "connected" message, right?). At first I thought this was odd, but I get it.

Reply:
right. I can add comments to make this more clear. I also want to be able to show the thread even if we disconnect (so we can see message history, disconnect reason), so the easiest thing I could think of was "is there any message queue to display, if so, display it"